src/views/default-view.ts

```diff
@@ -385,6 +386,49 @@ export class DefaultView<N extends INodeBase, E extends IEdgeBase> implements IO
     }
   };
 
+  mouseRightClicked = (event: PointerEvent) => {
+    // event.preventDefault();
+    const mousePoint = this.getCanvasMousePosition(event);
+    const simulationPoint = this._renderer.getSimulationPosition(mousePoint);
+
+    if (this._strategy.onMouseRightClick) {
+      const response = this._strategy.onMouseRightClick(this._graph, simulationPoint);
+      const subject = response.changedSubject;
+
+      
+      if (subject) {
+        if (isNode(subject)) {
+          this._events.emit(OrbEventType.NODE_RIGHT_CLICK, {
+            node: subject,
+            event,
+            localPoint: simulationPoint,
+            globalPoint: mousePoint,
+          });
+        }
+        if (isEdge(subject)) {
+          this._events.emit(OrbEventType.EDGE_RIGHT_CLICK, {
+            edge: subject,
+            event,
+            localPoint: simulationPoint,
+            globalPoint: mousePoint,
+          });
+        }
+      } else {
+        this._events.emit(OrbEventType.CANVAS_RIGHT_CLICK, {
+          event,
+          localPoint: simulationPoint,
+          globalPoint: mousePoint,
+        });
+      }
+
+      
+
+      // if (response.isStateChanged || response.changedSubject) {
+      //   this._renderer.render(this._graph);
+      // }
```

Comment:
```suggestion
      if (response.isStateChanged || response.changedSubject) {
        this._renderer.render(this._graph);
      }
```

Is there a particular reason why this is uncommented? The default click strategy you added selects a node or an edge which means its style will change and that will require a new render. That is the point of these lines.